addons/web/models/ir_http.py

```diff
@@ -102,6 +102,7 @@ def get_frontend_session_info(self):
             'profile_session': request.session.profile_session,
             'profile_collectors': request.session.profile_collectors,
             'profile_params': request.session.profile_params,
+            'show_effect': request.env['ir.config_parameter'].sudo().get_param('base_setup.show_effect'),
```

Comment:
is this relevant for public users?

Reply:
It could be if in we wanted to use effects visible to public users (currently there are none), but the frontend session is used even for logged in users when they are in the frontend, so it's relevant for them.